src/game_battlealgorithm.h

```diff
@@ -187,6 +186,15 @@ class AlgorithmBase {
 	 * @return Battle Animation or NULL if no animation is assigned
 	 */
 	const RPG::Animation* GetAnimation() const;
+	const RPG::Animation* GetSecondAnimation() const;
+
+	/**
+	* Checks if the animation has already played once
```

Comment:
That's a typical Albeleon lack-of-space doc-comment

Reply:
With lack-of-space I meant that there is a single space missing before the *, Albeleon did this the whole time.